src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/CreditorsAfterOneYearService.java

```diff
@@ -30,4 +31,14 @@ CreditorsAfterOneYear getCreditorsAfterOneYear(String transactionId, String comp
      */
     List<ValidationError> submitCreditorsAfterOneYear(String transactionId, String companyAccountsId, CreditorsAfterOneYear creditorsAfterOneYear)
             throws ServiceException;
+
+    /**
+     * Delete the creditors after one year note
+     *
+     * @param transactionId The id of the CHS transaction
+     * @param companyAccountsId The company accounts identifier
+     * @return A list of validation errors, or an empty array list if none are present
+     * @throws ServiceException if there's an error on deletion
+     */
+    List<ValidationError> deleteCreditorsAfterOneYear(String transactionId, String companyAccountsId) throws ServiceException;
```

Comment:
This method can be `void`. Delete's should never be 'bad requests'

Reply:
👍